modules/library.js

```diff
@@ -0,0 +1,57 @@
+import Storage from './storage.js';
+import Book from './book.js';
+
+export default class Library {
```

Comment:
Great job @kensteph 🤝

- [ ] [Optional] Kindly make sure the nav links work as expected and we are able to view the page where to add a book, and also display the contact message.
![awesome](https://user-images.githubusercontent.com/63917725/212756212-ee684ee1-90fd-4367-92d6-1db8663c3bca.PNG)

- [ ] [Optional] Kindly make sure we are able to add and remove a book from our list.